rancher/v1.2/en/quick-start-guide/index.md

```diff
@@ -79,7 +79,7 @@ If you click on the dropdown of the **_first-container_**, you will be able to p
 Rancher will display any containers on the host even if the container is created outside of the UI. Create a container in the host's shell terminal.
 
 ```bash
-$ docker run -d -it --name=second-container ubuntu:14.04.2
+$ docker run -it --name=second-container ubuntu:14.04.2
```

Comment:
I'd rather remove this line "If you terminate the container by exiting the shell, the Rancher UI will immediately show the stopped state of the container. and keep the `-d`